scripts/2022-10-01-wireguard-restructure.sh

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env bash
+
+# support user renaming of script
+SCRIPT=$(basename "$0")
+
+# useful function
+isContainerRunning() {
+   if STATUS=$(curl -s --unix-socket /var/run/docker.sock http://localhost/containers/$1/json | jq .State.Status) ; then
```

Comment:
It's possible that users may not have `jq` installed if using the old menu. `jq` could be added to the install section in menu.sh.